core/src/G3Timesample.cxx

```diff
@@ -222,6 +281,8 @@ PYBINDINGS("core")
           "consistency.  Raises ValueError if there are problems.")
 	.def("Concatenate", &G3TimesampleMap::Concatenate,
           "Concatenate two compatible G3TimesampleMap.")
+	.def("Sort", &G3TimesampleMap::Sort,
```

Comment:
Shouldn't this be sort() instead of Sort() in Python? It differs from the style of the other Python bindings, but sort() is a standard Python function so we probably gain net in terms of POLA.

Reply:
I'm fine with making it lowercase -- I'd also probably make `check` and `concatenate` lowercase in python too, since we already have a lowercase `concatenate` method for G3Timestreams.  @mhasself any opinion here?

Reply:
Lower case interface is fine with Me.  Ahem, with me.